test.js

```diff
@@ -1,23 +1,62 @@
-require('dotenv').load()
-const test = require('ava')
-const Deskbookers = require('./dist/index')
+import dotenv from 'dotenv'
```

Comment:
@vdvleon Ah, my bad. Forgot that. Don't worry about it, I'll take a look at tests